electricitymap/contrib/lib/models/events.py

```diff
@@ -24,6 +24,24 @@ def set_value(self, mode: str, value: Optional[float]) -> None:
         """
         self.__setattr__(mode, value)
 
+    def add_value(
```

Comment:
I'm wondering if it wouldn't be better to use the merge production outputs static method for this purpose. Though the PR for negative values is still on hold #5402.

Reply:
I think yes you might be right, the add method might be better suited here.